paper/main_sections/clustering.tex

```diff
@@ -56,3 +56,47 @@ \subsection{Hierarchical Clustering}
 to see the parts of the brain that have the strongest relationship with the
 signal and compare them to the results of the origin research paper.
 
+\subsection{t-Statisics Grouping}
+
+\par We use the t-statistics that we found earlier for another type of cutoff
+analysis. The t-statistics measure the size of the difference relative to the 
+variation in the data. Although the t-statistics go hand in hand with the 
+Benjamini-Hochberg analysis on the ``p-values'', we ultimately decided that an 
+additional process of selecting t-statistics based on a threshold was 
+necessary because the ``p-values'' assume a normal distribution while the 
+t-statistics do not assume this from the data.
+
+\par Since the magnitude from zero represents how significant the test is for
+t-statistics, we collected the absolute value t-statistics that were above a 
+certain threshold. This was the opposite of the procedure for finding the 
+significant tests in the Benjamini-Hochberg process. The threshold, also called
```

Comment:
Could you say this in a different way, to reflect that we also want p-values small <=> large abs(t) values.  Also maybe throw in the fact that p-values are related to abs(t) with a 1-to-1 (injective) function. (maybe even write out the function).